### ISSUE TITLE
Field name and options should be normalized to A-Z+0-9

### ISSUE THREAD

**Issue by FelixMalfait** [twentyhq/twenty#3296]
I tried to create a field "industry" on the company object. 
One of the options had emoji in its value:
<img width="434" alt="Screenshot 2024-01-08 at 12 15 35" src="https://github.com/twentyhq/twenty/assets/6399865/f50fdd2a-0916-4484-937e-53ff2d717292">

But now my workspace is broken and I get this error everywhere:
<img width="791" alt="Screenshot 2024-01-08 at 12 04 54" src="https://github.com/twentyhq/twenty/assets/6399865/1655fd36-6b22-4514-9782-9e087b29352d">

Related: #3066 #2146

**Comment by AdityaPimpalkar** [comment]
Hey @FelixMalfait I encountered a similar issue in #3270. 

It looks like while create options, we are assigning `value` property as the `Name` of enum. This error is thrown because GraphQL has internal validation on GraphQLEnumType. I did a workaround from the front end to solve this, but I guess that shouldn't be long term solution.

**Comment by magrinj** [comment]
@AdityaPimpalkar You're right the problem is actually also related to #3270, but in that case the name inside the enum are chosen by the user. We should change the regex to only accept `[a-zA-Z]+`, and drop number and emoji support
